### DISCUSSION TITLE
New Plugin: beets-lms

### DISCUSSION THREAD

**Discussion by InvisibleFunction** [beetbox/beets#5509] [General]
Hi all, I just released a new beets plugin for those using Lyrion Music Server (aka LMS, fka Logitech Media Server). I got tired of importing new music and then walking to my stereo only to find I needed to manually kick off a rescan and wait for it to complete before I could play my new music.

This plugin starts a rescan following a beets import and also includes a few subcommands for interacting with LMS.

https://github.com/InvisibleFunction/beets-lms

I'd love feedback if you've got ideas about what other useful things it could do to integrate with LMS. I'd also love feedback if you have beets plugin experience and have ideas for improvement!

**Comment by mirthestam** [comment]
This is interesting! I’m currently using a file-system watcher plugin, but this seems like a better solution. Thanks for your work!
